packages/@aws-cdk/aws-apigateway/test/test.method.ts

```diff
@@ -301,6 +301,63 @@ export = {
 
     // THEN
     test.throws(() => api.root.addMethod('GET', integration), /cannot set 'vpcLink' where 'connectionType' is INTERNET/);
+    test.done();
+  },
+
+  'methodResponse set one or more method responses via options'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigateway.RestApi(stack, 'test-api', { deploy: false });
+
+    // WHEN
+    new apigateway.Method(stack, 'method-man', {
+      httpMethod: 'GET',
+      resource: api.root,
+      options: {
+        methodResponses: [{
+            statusCode: '200'
+          }, {
+            statusCode: "400",
+            responseParameters: {
+              'method.response.header.killerbees': false
+            }
+          }, {
+            statusCode: "500",
+            responseParameters: {
+              'method.response.header.errthing': true
+            },
+            responseModels: {
+              'application/json': 'Empty',
+              'text/plain': 'Empty'
```

Comment:
Do you have a use-case where you'll be hard-coding the model names or creating them as part of your stack?

Reply:
I don't. My current use-case is just using the default Empty model. Maybe this is worth at least adding an `IModel` interface, as @RomainMuller mentioned below, and perhaps having some default Empty implementation.